server/client.ts

```diff
@@ -96,7 +96,7 @@ class Client {
 		[socketId: string]: {token: string; openChannel: number};
 	};
 	config!: UserConfig & {
-		networks?: Network[];
+		networks?: NetworkConfig[];
```

Comment:
Do you know why this `networks` isn't specified in `UserConfig` directly?

Reply:
I assume it's optional in the attribute because of:

https://github.com/thelounge/thelounge/blob/efd24fd12cad9192d6f333c5a3c01c33ad23b0c6/server/client.ts#L182-L184

but required as the constructor argument.

but I don't see the point of making it required. Fixed